cmd/snap-bootstrap/initramfs_systemd_mount.go

```diff
@@ -443,12 +443,16 @@ func writeInitramfsMountUnit(what, where string, utype unitType) error {
 	return os.Symlink(filepath.Join("..", unitFileName), linkPath)
 }
 
-func assembleSysrootMountUnitContent(what, mntType string) string {
+func assembleSysrootMountUnitContent(what, mntType string, opts []string) string {
 	var typ string
+	optionStr := ""
 	if mntType == "" {
 		typ = "Type=none\nOptions=bind"
 	} else {
 		typ = fmt.Sprintf("Type=%s", mntType)
+		if len(opts) != 0 {
+			optionStr = "\nOptions=" + strings.Join(opts, ",")
+		}
```

Comment:
I'd prefer to maybe now append the "bind" option if case the type is none, to make it more generic and allow additional options also in that case.